app/helpers/admin/categories_helper.rb

```diff
@@ -3,8 +3,12 @@ def root_categories
     @root_categories = Category.root_categories.pluck :name, :id
   end
 
-  def get_name_root_category id
-    return @root_category = Category.find_by(id: id).name if id.present?
+  def parent_categories
+    @parent_categories = Category.parent_categories.pluck :name, :id
+  end
+
+  def get_name_category id
+    return @name_category = Category.find_by(id: id).name if id.present?
```

Comment:
```suggestion
    return @name_category = Category.find_by(id: id)&.name if id.present?
```
ri chứ, lỡ find ko ra

Reply:
Dạ vâng ạ